application/config.json.template

```diff
@@ -87,9 +87,9 @@
         "wsh"
     ],
     "logInfoChannelWebhook": "<put_your_webhook_here>",
-    "logErrorChannelWebhook": "<put_your_webhook_here>"
+    "logErrorChannelWebhook": "<put_your_webhook_here>",
     "starboard": {
-        "emojiNames" : ["oof", "lmao"]
+        "emojiNames" : ["oof", "lmao"],
         "channelName": "oofs_and_lmaos"
```

Comment:
can u as default use sth thats available on any discord server by default? id just go for `star` and call the channel `starboard`.